### ISSUE TITLE
Visualise geospatial data on dataset details page

### ISSUE THREAD

**Issue by CharliePatterson** [digital-land/submit#387]
## Context:

- Displaying geospatial data on a map enables LPAs to visually check that the data they've provided is correct. A key finding from the Alpha phase was that this approach had much better responses that just displaying the data in a list or table format, as they'd be able to visually check the boundaries and locations were correct.

## Acceptance criteria:

- As a data provider, when I'm viewing the details page of a dataset that contains geospatial data, I can see that data displayed on an interactive map

## Designs:

![Image](https://github.com/user-attachments/assets/984e2d03-9ee3-4850-b6e8-fa4fb61f921c)

------
### if a dataset has multiple endpoints:

![Image](https://github.com/user-attachments/assets/f62924b7-be26-4cca-b2df-6e052be29adf)

**Comment by DilwoarH** [comment]
@CharliePatterson - there are some datasets which have so many points that the API returns an error (query taking too long).

We will need to look at using layers for these as suggested by @GeorgeGoodall.

**Comment by CharliePatterson** [comment]
@DilwoarH would this be resolved when we switch over to using the performance db?

**Comment by DilwoarH** [comment]
> @DilwoarH would this be resolved when we switch over to using the performance db?

I don't think so as that would still use Datasette (which has the limitation).

The map layers work better long term as they are generated and cached - therefore won't require additional processing on our end. We need to investigate the following:
- Are the cached layers at a LPA level
- Which datasets are cached

Hope this makes sense
